### PR TITLE
RN CNV Bug 1835426 VMI name in the template

### PR DESCRIPTION
**Pull request by lmandavi**
This PR is associated with https://bugzilla.redhat.com/show_bug.cgi?id=1835426.

### CODE REVIEW

**Comment by lmandavi** [comment]
@rmohr - This PR is ready for review. Adding @aburdenthehand for awareness.

File: virt/virt-2-4-release-notes.adoc [hunk 1]
```diff
@@ -100,13 +100,17 @@ from being inadvertently started before an upload has completed.
...
-//https://bugzilla.redhat.com/show_bug.cgi?id=1835426 - Requested doc text for this defect.
+
+* If a name is specified for a virtual machine instance (VMI) in the VMI template for the virtual machine (VM), it is ignored. The names for the VMI and VM are matched.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1835426[*BZ#1835426*])
 
 * KubeMacPool is disabled in {VirtProductName} {VirtVersion}. This means that a secondary
 interface of a Pod or virtual machine obtains a randomly generated MAC address rather
 than a unique one from a pool. Although rare, randomly assigned MAC addresses can conflict.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1816971[*BZ#1816971*])
 
+*
```
Comment (rmohr) [inline virt/virt-2-4-release-notes.adoc:111]: That was probably not intended?

File: virt/virt-2-4-release-notes.adoc [hunk 2]
```diff
@@ -100,13 +100,17 @@ from being inadvertently started before an upload has completed.
 [id="virt-2-4-known-issues"]
 == Known issues
 //https://bugzilla.redhat.com/show_bug.cgi?id=1855067 - Requested doc text for this defect.
-//https://bugzilla.redhat.com/show_bug.cgi?id=1835426 - Requested doc text for this defect.
+
+* If a name is specified for a virtual machine instance (VMI) in the VMI template for the virtual machine (VM), it is ignored. The names for the VMI and VM are matched.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1835426[*BZ#1835426*])
```
Comment (rmohr) [inline virt/virt-2-4-release-notes.adoc:104]: how about `it is now correctly ignored`, to indicate that this is what we want?

File: virt/virt-2-4-release-notes.adoc (hunk 1 above)
Comment (lmandavi) [reply virt/virt-2-4-release-notes.adoc:111]: This content is not part of this PR.

File: virt/virt-2-4-release-notes.adoc (hunk 2 above)
Comment (lmandavi) [reply virt/virt-2-4-release-notes.adoc]: Reshaped the wording to indicate the behavior applies to this release. Please review.

**Comment by lmandavi** [comment]
@rmohr Please review.

File: virt/virt-2-4-release-notes.adoc [hunk 3]
```diff
@@ -109,10 +109,14 @@ from being inadvertently started before an upload has completed.
...
 == Known issues
 //https://bugzilla.redhat.com/show_bug.cgi?id=1855067 - Requested doc text for this defect.
-//https://bugzilla.redhat.com/show_bug.cgi?id=1835426 - Requested doc text for this defect.
-//https://bugzilla.redhat.com/show_bug.cig?=id=1959237
+
 * If you upgrade to {VirtProductName} {VirtVersion}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, you must use the newer version of the template. Disregard the older version to avoid issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1859237[*BZ#1859237*])
 
+* In this release, if a name is specified for a virtual machine instance (VMI) in the VMI template associated with the virtual machine (VM), it is ignored. The names for the VMI and VM are matched correctly.
+
+If a name is specified for a virtual machine instance (VMI) in the VMI template for the virtual machine (VM), it is ignored. The
```
Comment (rmohr) [inline virt/virt-2-4-release-notes.adoc:117]: Is this sentence  intentionally there too? The sentence above looks descriptive enough. Maybe move just the link up and remove the rest.

Comment (lmandavi) [reply virt/virt-2-4-release-notes.adoc:117]: @rmohr The sentences on Line 117 and 188 are redundant, removing them.

**Comment by lmandavi** [comment]
Screenshot from 2020-07-27 11-44-43

**Comment by lmandavi** [comment]
@rmohr - Updated the defect notes. The behavior works as intended, therefore this does not fall into the classification of known issue defects. Removing this PR info from the release notes. Adding @aburdenthehand and @vikram-redhat for awareness.

**Comment by lmandavi** [comment]
Resolved merge conflict and pushed.

File: virt/virt-2-4-release-notes.adoc [hunk 4]
```diff
@@ -195,5 +192,10 @@ nodes.
 running either `oc adm drain` or `kubectl drain`. Do not run these commands on
 the nodes of any clusters where {VirtProductName} is deployed. The nodes might not
 drain if there are virtual machines running on top of them.
+<<<<<<< HEAD
```
Comment (kalexand-rh) [inline virt/virt-2-4-release-notes.adoc:195]: @lmandavi, please fix this merge conflict and squash.

And are you sure you're changing the link again?

Comment (vikram-redhat) [inline virt/virt-2-4-release-notes.adoc:195]: @lmandavi - this hasn't been cleanly fixed.

Comment (lmandavi) [reply virt/virt-2-4-release-notes.adoc:195]: @vikram-redhat @kalexand-rh I am looking at this again.

Comment (lmandavi) [reply virt/virt-2-4-release-notes.adoc:195]: @kalexand-rh - this particular content shown here is not related to this PR, it was content belonging to another team member. I need to go back into the defect and see if this content is relevant and should be retained or deleted.

Comment (aburdenthehand) [reply virt/virt-2-4-release-notes.adoc:195]: @lmandavi Should we just close this PR? The release note it was adding has been nixed. Functionally now it is only removing some placeholder comments (once the conflict is resolved)

Comment (kalexand-rh) [reply virt/virt-2-4-release-notes.adoc]: +1 to just closing the PR.

**Comment by lmandavi** [comment]
@kalexand-rh - please check. commits were squashed.

File: virt/virt-2-4-release-notes.adoc (hunk 4 above)
Comment (lmandavi) [reply virt/virt-2-4-release-notes.adoc]: OK. Closing this PR.

**Comment by lmandavi** [comment]
Closing this PR.

**Pull request closed without merging**